src/main/java/org/folio/service/transformationfields/TransformationFieldsServiceImpl.java

```diff
@@ -52,6 +70,50 @@
     return promise.future();
   }
 
+  @Override
+  public Future<Void> validateTransformations(List<Transformations> transformations) {
```

Comment:
Also please add some docs here, as to what validations this method does, for readability

Reply:
I added Javadoc, thank you